manifests/g/GnuPG/GnuPG/devel/2.5.5/GnuPG.GnuPG.devel.installer.yaml

```
@@ -0,0 +1,22 @@
+# Created with komac v2.10.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GnuPG.GnuPG.devel
+PackageVersion: 2.5.5
+InstallerLocale: en-US
+InstallerType: nullsoft
+Scope: machine
+UpgradeBehavior: install
+ProductCode: GnuPG
+ReleaseDate: 2025-03-07
+AppsAndFeaturesEntries:
+- DisplayName: GNU Privacy Guard
+  ProductCode: GnuPG

```

```suggestion
ReleaseDate: 2025-03-07
AppsAndFeaturesEntries:
- DisplayName: GNU Privacy Guard
```
@SpecterShell 
> Adding this package would prevent the stable one from being updated by WinGet as they have the same `ProductCode`
  
Would this still be an issue if the `ProductCode` field gets omitted from .devel versions?